docs/current_docs/api/remote-modules.mdx

```diff
@@ -37,7 +37,36 @@ dagger -m git@github.com:kpenfound/dagger-modules/golang@v0.2.0 call \
   export --path=./build
 ```
 
-## Configuring SSH authentication
+## Authentication Methods
+
+Dagger supports both HTTPS and SSH authentication for accessing remote repositories.
+
+### HTTPS Authentication
+
+For HTTPS authentication, Dagger uses your system's configured Git credential manager. This means if you're already authenticated with your Git provider, Dagger will automatically use these credentials when needed.
+
+The following credential helpers are supported:
+- Git Credential Manager
+- macOS Keychain
+- Windows Credential Manager
+- Custom credential helpers configured in your `.gitconfig`
+
+To verify your credentials are properly configured, try cloning a private repository:
+
+```shell
+git clone https://github.com/username/private-repo.git
+```
+
+If this works, Dagger will be able to use the same credentials to access your private repositories.
+
+#### Common Git Credential Manager Setups
+
+- **GitHub**: Use [`gh auth login`](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git#github-cli) or [configure credentials via `git-credential-manager`](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git#git-credential-manager)
```

Comment:
```suggestion
- GitHub: Use [`gh auth login`](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git#github-cli) or [configure credentials via `git-credential-manager`](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git#git-credential-manager)
```

We try to avoid using bold/italic formatting.